### DISCUSSION TITLE
I installed the Docker version of AC, and I need to replace the 2 .mp3 installed

### DISCUSSION THREAD

**Discussion by pmgreb** [AzuraCast/AzuraCast#4708] [Q&A]
Hello, I need to replace the two mp3 files installed:

resources/error.mp3
resources/30_second_promo.mp3

I tried searching in the dockers containers, but I'm exhausted, I can't find which container and where are those files located. Anyone that can enlighten me? Thanks!

**Comment by BusterNeece** [comment]
@pmgreb Replacing the `error.mp3` file in that folder won't actually switch out the default error sound on most stations; that's something that requires following separate instructions: https://docs.azuracast.com/en/administration/docker#custom-default-track

As for replacing the 30-second promo, that's only ever linked to from our own documentation and never actually included into playlists or elsewhere in the system, so there should be no need to replace it on a per-installation basis.
